vars/executeTests.groovy

```diff
@@ -20,7 +20,7 @@ def call(Map<String, String> config = [:]) {
     def configData = readJSON text: env.configJSON
 
     // If we have a repo defined to checkout the infra configuration playbooks from, let's handle that
-    if (configData.tests.repo) {
+    if (configData?.tests?.repo) {
```

Comment:
This could still result in a NPE if the configData was null. I've offered a suggestion above to deal with that, and this could be modified to:

```suggestion
    if (configData.tests?.repo) {
```